Readme.md

```diff
@@ -1 +1,62 @@
-## Webchat examples with different frameworks
\ No newline at end of file
+# Helpshift web chat examples
+
+This document serves as a guide to understand the purpose of this repository, the frameworks covered, how to navigate the repository and how to contribute
+
```

Comment:
No need for these lines. A README, by convention is intended to do exactly this.